plugins/auth/ldap/auth.go

```diff
@@ -35,6 +35,7 @@ type AuthLDAP struct {
 		ValidateHostname string `yaml:"validate_hostname"`
 		AllowInsecure    bool   `yaml:"allow_insecure"`
 	} `yaml:"tls_config"`
+	MFA		[]plugins.MFAConfig `yaml:"mfa"`
```

Comment:
There is no user-specific MFA configuration which means the **same** configuration will be applied for all users. Though this might work for DUO this definitely breaks all other MFA providers.

Reply:
Additionally this removes the possibility to have specific users not being configured with MFA (bot users for example).